msm/src/circuit_design/constraints.rs

```diff
@@ -56,6 +55,16 @@ impl<F: PrimeField, CIx: ColumnIndexer, LT: LookupTableID> ColAccessCap<F, CIx>
         Expr::Atom(ExprInner::Constant(cst_expr_inner))
     }
 }
+impl<F: PrimeField, CIx: ColumnIndexer, LT: LookupTableID> NextCap<F, CIx>
```

Comment:
You have to implement lenses for your new capability too (missing in this PR), otherwise it won't compose well in the future.